### DISCUSSION TITLE
Rules for Styleshets Tiddler and TW 5.3.x

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#8279] [Q&A]
See https://tiddlywiki.com/prerelease/#Pragma%3A%20%5Crules
Normally when one uses a tiddler of type `text/vnd.tiddlywiki` uses the blow rule as recommended by official documentation

```
\rules only filteredtranscludeinline transcludeinline macrodef macrocallinline
```

With advent of new features like procedures and functions, should we upgrade the rules to cover these new features which may be used in a `text/vnd.tiddlywiki`  stylesheet tiddler?

**Comment by pmario** [comment]
When we change the stylesheets \define macros, we can also change those rules.
